packages/mui-material/src/Tab/Tab.js

```diff
@@ -10,11 +10,13 @@ import unsupportedProp from '../utils/unsupportedProp';
 import tabClasses, { getTabUtilityClass } from './tabClasses';
 
 const useUtilityClasses = (ownerState) => {
-  const { classes, textColor, fullWidth, wrapped, icon, label, selected, disabled } = ownerState;
+  const { classes, className, textColor, fullWidth, wrapped, icon, label, selected, disabled } =
+    ownerState;
 
   const slots = {
     root: [
       'root',
+      'className',
```

Comment:
```suggestion
```

This will generate class name: `MuiTab-className`

Reply:
> This will generate class name: MuiTab-className  
Sorry my implementation was naive, obviousy we dont want `MuiTab-className`.